### DISCUSSION TITLE
Same script but evaluating time increase.

### DISCUSSION THREAD

**Discussion by KebingXUE** [PaddlePaddle/PaddleSpatial#270] [BaiduKDDCup2022] [answered]
Dear organizers:

I submitted twice my script with just the model states different, the first my script pass the evaluation, and the second time it failed due to the time limitation. Is there a lack of computation ability?

**Comment by ramber1836** [comment, marked as answer]
Did your second submission fail with the Pending Timeout Error?

**Reply by KebingXUE** [reply]
I submitted 3 times today, first time it passed, second time I change the model state file without any other changes it failed with Pending TIme Error. Third time I optimized my script so it requires half the time compared with my first two submission(I estimated the running time locally), the last submission passed but still takes about 1h30m, so I thought maybe there is some perturbations on the competetion server.
